rita/Cargo.toml

```diff
@@ -6,11 +6,19 @@ authors = ["Jehan <jehan.tremback@gmail.com>"]
 [dependencies]
 babel_monitor = { path = "../babel_monitor" }
 althea_kernel_interface = { path = "../althea_kernel_interface" }
-#traffic_watcher = { path = "./traffic_watcher"} # PLACEHOLDER, CHANGE/REMOVIE UPON MERGE
+traffic_watcher = { path = "./traffic_watcher"}
 payment_controller = { path = "./payment_controller" }
 debt_keeper = { path = "./debt_keeper" }
-
+tunnel_manager = { path = "./tunnel_manager" }
+num256 = { path = "../num256" }
 ip_network = "0.1"
 log = "0.3"
 docopt = "0.8.1"
 simple_logger = "0.4.0"
+rouille = "1.0.3"
+serde = "1.0.24"
+serde_derive = "1.0.24"
+serde_json = "1.0.8"
+reqwest = "0.8.2"
```

Comment:
This is going to cause that OpenSSL stench to return

Reply:
Do we need HTTP this bad?

Reply:
It's an easy to use protocol that is understood by everybody already, which is why I'm using it. I agree that the libraries that implement it tend to be heavy.  We could in principle use any rpc technique.

I can take a look at how hard it is to submit a patch to our http stuff to put https behind a cfg flag.

Reply:
Sorry, I missed most of this stuff in the chat. What are the specific issues around openssl? This will help inform my pull request.

Reply:
Actually, thinking about it more. We could avoid https on node-to-node comms, but as soon as we're talking to servers on the internet we will either need to use https or reinvent the wheel.

Reply:
Yeah, we're probably gonna hit that brick wall anyway when we'll need to reach APIs like an automated version of the node locator map API